bundler/lib/bundler/cli.rb

```diff
@@ -544,6 +544,7 @@ def viz
     method_option :ci, type: :string, lazy_default: Bundler.settings["gem.ci"] || "", enum: %w[github gitlab circle], desc: "Generate CI configuration, either GitHub Actions, GitLab CI or CircleCI. Set a default with `bundle config set --global gem.ci (github|gitlab|circle)`"
     method_option :linter, type: :string, lazy_default: Bundler.settings["gem.linter"] || "", enum: %w[rubocop standard], desc: "Add a linter and code formatter, either RuboCop or Standard. Set a default with `bundle config set --global gem.linter (rubocop|standard)`"
     method_option :github_username, type: :string, default: Bundler.settings["gem.github_username"], banner: "Set your username on GitHub", desc: "Fill in GitHub username on README so that you don't have to do it manually. Set a default with `bundle config set --global gem.github_username <your_username>`."
+    method_option :install, type: :boolean, default: true, desc: "Automatically run `bundle install` after creation."
```

Comment:
```suggestion
    method_option :install, type: :boolean, default: Bundler.settings["gem.install"], desc: "Automatically run `bundle install` after creation."
```

Otherwise it will run `bundle install` every time even if you configure `bundle config gem.install false`.

To cover the fix, you could also add one more test with `BUNDLE_GEM__INSTALL: false` that checks that the message is not generated when `bundle gem` run without `--install` or `--no-install` flags.